src/workerd/jsg/jsvalue.h

```diff
@@ -325,13 +325,21 @@ class JsProxy final: public JsBase<v8::Proxy, JsProxy> {
 
 V(Symbol)
 V(BigInt)
-V(Number)
 V(Int32)
 V(Uint32)
 V(Set)
 
 #undef V
 
+class JsNumber final: public JsBase<v8::Number, JsNumber> {
+ public:
+  kj::Maybe<double> value(Lock& js) const KJ_WARN_UNUSED_RESULT;
+  bool isSafeInteger(Lock& js) const KJ_WARN_UNUSED_RESULT;
+  kj::Maybe<double> toSafeInteger(Lock& js) const KJ_WARN_UNUSED_RESULT;
+
+  using JsBase<v8::Number, JsNumber>::JsBase;
+};
```

Comment:
The definition of this should be moved into jsvalue.h, just like the others like `JsBoolean`

Reply:
what if i told you, this file is jsvalue.h?

Reply:
Ha!! Wow... I totally misread the header name and could have sworn it said `jsg.h` ... sigh. I think that means I get to take the rest of the week off? Right? Right? ...... Right?